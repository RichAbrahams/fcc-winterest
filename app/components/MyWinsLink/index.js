/**
*
* MyWinsLink
*
*/

import React from 'react';
import { browserHistory } from 'react-router';
import { FA } from './styles';

function MyWinsLink() {
  return (
    <FA
      name="user"
      size="4x"
      onClick={() => browserHistory.push('/mywins')}
    />
  );
}

MyWinsLink.propTypes = {

};

export default MyWinsLink;
