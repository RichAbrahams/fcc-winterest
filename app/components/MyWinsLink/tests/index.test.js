import React from 'react';
import renderer from 'react-test-renderer';
import MyWinsLink from '../index';

it('renders correctly', () => {
  const tree = renderer.create(
    <MyWinsLink />).toJSON();
  expect(tree).toMatchSnapshot();
});
