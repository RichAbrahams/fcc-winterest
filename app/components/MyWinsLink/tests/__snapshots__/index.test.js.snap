exports[`test renders correctly 1`] = `
<span
  aria-hidden={true}
  className="fa fa-user fa-4x VmiKC"
  onClick={[Function]} />
`;
